pallets/alt-vr/src/tests/extrinsics.rs

```diff
@@ -0,0 +1,130 @@
+use super::mock::*;
+use crate::tests::mock;
+use crate::{AltvrData, Error};
+use frame_support::{assert_noop, assert_ok};
+use frame_system::RawOrigin;
+
+#[test]
+fn create_altvr_happy() {
```

Comment:
Don't forget to update the name of test.

Reply:
done